homeassistant/components/unifi/device_tracker.py

```diff
@@ -40,6 +40,7 @@
     "is_guest",
     "noted",
     "qos_policy_applied",
+    "signal",
```

Comment:
This should not be added as an attribute. It will cause a lot of state changes.

Reply:
What would be an alternative approach? Separate sensor(s) that you can add via the integration configuration flow?

Reply:
> Could also add a configuration step where you explicitly choose which attributes you want to add.

We used to call that monitored conditions, which we try to get rid off 😉 


> What would be an alternative approach? Separate sensor(s) that you can add via the integration configuration flow?

Separate sensors that are coupled with the device it belongs to, disabled by default in the entity registry on creation. That way, a user can choose to activate them.

Reply:
> Separate sensors that are coupled with the device it belongs to, disabled by default in the entity registry on creation. That way, a user can choose to activate them.

The current flow only creates them when enabled afaik: 
![image](https://user-images.githubusercontent.com/547021/112657708-f98a8780-8e52-11eb-9d51-03fe7e43515b.png)

If extending that is acceptable I'll work on it. If we need to change that flow as you mentioned by creating the sensors in a disabled state to allow individual control, I'm afraid it'll be too complicated for the time I'm able to spend on it.

Let me know if the existing flow can be extended to include a signal sensor.

Reply:
Yes, a new option is the way to go here

Reply:
And a new sensor class with test coverage, should probably be mostly copy paste work for you